CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.90.1
+
+### Bug fixes
+
+* Making HandlerController#emptyAsyncRealmObject and HandlerController#realmObjects thread safe (#2724).
```

Comment:
Referencing internal API implementations doesn't mean anything to users. Could this be rephrased in a way so users know how it could have impacted them instead.